multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1421,6 +1422,9 @@ def _top_k_average(
         strict_loading=True,
         standalone=True,
     ):
+        if self._problem_type is not None and self._problem_type == NER:
+            validation_metric_name = OVERALL_F1  # seqeval only support overall_f1
```

Comment:
There is no need to add the additional "if" here. If the problem is NER, the proper validation_metric and eval_metric should be set here: https://github.com/autogluon/autogluon/blob/10b39aceb35e943f7c62f095f0005b5b42aa8aa4/multimodal/src/autogluon/multimodal/utils/metric.py#L104-L105

Reply:
> There is no need to add the additional "if" here. If the problem is NER, the proper validation_metric and eval_metric should be set here:
> 
> https://github.com/autogluon/autogluon/blob/10b39aceb35e943f7c62f095f0005b5b42aa8aa4/multimodal/src/autogluon/multimodal/utils/metric.py#L104-L105

in line 1475, best_score = self.evaluate(val_df, [validation_metric_name])[validation_metric_name], the evaluate function is called and validation_metric_name is used. However, in the evaluate function, seqeval is used and only overall_f1, overall_accuracy, overall_precision, overall_recall, and tag name are valid evaluation metric. NER_TOKEN_F1 is not supported, that is why I used OVERALL_F1 as validation_metric_name and eval_metric_name. To separate them, we have to do this trick here.

Reply:
Updated that we added a `#FIXME` in the source code. So I'm resolving this and we will revisit later.